src/app/app.component.html

```diff
@@ -6,40 +6,45 @@ <h2>Unsupported Browser</h2>
     </div>
   </div>
 }
-@if (currentUrlWithoutHash$ | async; as currentUrl) {
-  <div class="skip-to-content-link">
-    <!--
-    Can't use a simple relative link for the URL because we have base href set of /d/ set in non-dev environments
-    which is needed for links to assets but would break this.
-    -->
-    <a [href]="currentUrl.toString() + '#content'">Skip to content</a>
-  </div>
-}
 
-@if (isDataLoaded) {
-  <biggive-main-menu
-    [donateUrlPrefix]="donateUriPrefix"
-    [blogUrlPrefix]="blogUriPrefix"
-    [experienceUrlPrefix]="experienceUriPrefix"
-    [isLoggedIn]="isLoggedIn"
-    [smallCharityWeekEnabled]="true"
-    [someCampaignHasHomePageRedirect]="someCampaignHasHomePageRedirect()"
-  ></biggive-main-menu>
-}
+<div role="banner" aria-label="Menu &amp; skip">
```

Comment:
The aim with this and similar containers below is to quickly fix errors from the skip link etc. not being inside a landmark. I didn't want to create nested `navigation` role landmarks so a catch-all of `banner` seemed the best quick win available.

Reply:
Looks good, I didn't know much about landmarks but looks like you're following this rule https://dequeuniversity.com/rules/axe/4.1/region 

Would it make sense to use a `<header>` instead of `<div>`? 

Not sure if the aria-label is accessary, I guess menu & skip is what people expect to find inside the banner anyway, might just add to listening time?

Reply:
I think some kind of distinguishing label is compulsory when there is more than one of the same landmark on the page, which there will be for `banner`.

Using `<header>` instead sounds like a good idea, changed in 316fb1e33623adbd25d95211d9af8334276594c4